src/css/vocabulary.css

```diff
@@ -1407,6 +1407,39 @@ body > footer .subscribe form input.button {
     color: rgb(255, 255, 255);
 }
 
+input:focus {
+    outline: none; /* Remove default focus outline */
+}
+
+.input.invalid {
+    border-color: red;
+}
+
+.input.valid {
+    border-color: green;
+}
+
+.input-wrap {
+    display: flex; 
+    align-items: center; 
+}
+
+.email {
+    flex-grow: 1; 
+    margin-right: 0px; 
+}
+
+.error-message {
+    font-size: 12px; 
+    color: red; 
+    margin-top: 10px; 
+    display: block; 
+}
+.the-form {
```

Comment:
a class selector here isn't needed. I would suggest something that follows the progression of naming that accounts for specificity, such as: `.subscribe form`.

This would also mean you don't need the class in the HTML markup.